### ISSUE TITLE
analog sticks problem

### ISSUE THREAD

**Issue by Jennylinn** [Ryochan7/DS4Windows#36]
hello, its my first time to use DS4 tool i got new ps4 controller to use it with it 
all buttons works fine excepts left and right analog both pressing buttons by them self  or sometimes works in windows without auto pressing but my problem that all buttons works in game excepts analogs
plz help me what i have done wrong
sys windows 10
thank you in advance.

**Comment by Ryochan7** [comment]
Are you running the latest version (1.4.69)? There was an issue a couple of releases ago with mappings not working properly for the analog sticks. I have tested the various mapping options for the analog sticks in the current release and I didn't find any problems. Also, if you are using an edited profile, can you post your profile here?

**Comment by Jennylinn** [comment]
yes using the latest version still pressing buttons by it self 
here is my profile [https://ufile.io/wlbbq](url)

**Comment by Ryochan7** [comment]
Since you are using KB emulation for the sticks controls, you need to have a dead zone assigned for LS and RS or the associated keys will almost always be active. Lately I have been using a 9% dead zone for the analog sticks.

**Comment by Jennylinn** [comment]
ok i will try that

**Comment by Jennylinn** [comment]
thank you it works 👍  can you tell me how to
 make 2 buttons in one key like when i press square it press ctrl+1
i have used xpadder before, like setting circle  button by press 2 repeatedly and just pressing circle to turn it on and off can i do that in DS4 ?

**Comment by Ryochan7** [comment]
The macro system should allow for assigning ctrl+1 to a single button. I am not sure what kind of mapping you would be talking about for the second part. There is a toggle mode for actions but I am not sure that there is any way to repeatedly alternate between a key press and a key release; I tested the closest equivalent and it doesn't seem to work.

I would still use other programs for more complex KB+M emulation. Only a little work has gone into improving the KB+M emulation in DS4Windows.

Also, this ticket has shown a problem with dual inputs when both a key assignment and an xinput binding would be activated simultaneously. That problem was a minor oversight in the old code and it has been fixed.
